apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/index.tsx

```diff
@@ -24,23 +24,25 @@ import {
   useCurrentProject,
   useLocalStorage,
 } from '@latitude-data/web-ui'
+import type { StreamTextOutputAction } from '$/actions/documents/streamTextAction'
 import useDocumentVersions from '$/stores/documentVersions'
 import { useDebouncedCallback } from 'use-debounce'
 
 import { Header } from './Header'
 import Playground from './Playground'
 
 export const DocumentEditorContext = createContext<
-  { runAction: Function; streamTextAction: Function } | undefined
+  | {
+      streamTextAction: StreamTextOutputAction
+    }
+  | undefined
```

Comment:
Are these changes on top of #97 ? Feels like it.

Reply:
yup, talked offline. Gonna merge this, feel free to close #97